hbase-http/src/main/java/org/apache/hadoop/hbase/http/prometheus/PrometheusHadoopServlet.java

```diff
@@ -57,13 +60,19 @@ static String toPrometheusName(String metricRecordName, String metricName) {
    */
   @RestrictedApi(explanation = "Should only be called in tests or self", link = "",
       allowedOnPath = ".*/src/test/.*|.*/PrometheusHadoopServlet\\.java")
-  void writeMetrics(Writer writer) throws IOException {
+  void writeMetrics(Writer writer, boolean desc) throws IOException {
     Collection<MetricsRecord> metricRecords = MetricsExportHelper.export();
     for (MetricsRecord metricsRecord : metricRecords) {
       for (AbstractMetric metrics : metricsRecord.metrics()) {
         if (metrics.type() == MetricType.COUNTER || metrics.type() == MetricType.GAUGE) {
 
           String key = toPrometheusName(metricsRecord.name(), metrics.name());
+
+          if (desc) {
+            String description = metrics.description();
+            if (!description.isEmpty()) writer.append("# HELP ").append(description).append("\n");
```

Comment:
When you append a single char to the writer you can use the append(char c) instead if of append(CharSequence csq). I mean
`.append('\n'`)
instead of 
`.append("\n")`
Doesn't make much difference but slightly cheaper..

Reply:
I fixed it, thanks for the info!